### PR TITLE
Edit domain_adaptation for skimage support (#752)

### PR DESCRIPTION
**Pull request by hhawary**
1. Add support to histogram_matching in skimage version less than 0.16 related to issue (#752)
1. Following black format

### CODE REVIEW

**Review by Dipet — APPROVED** [review]

**Comment by ternaus** [comment]
What does this change do?

We do not use `histogram_matching` in the rest of the file.

File: albumentations/augmentations/domain_adaptation.py [hunk 1]
```diff
@@ -7,7 +7,10 @@
 from ..core.transforms_interface import ImageOnlyTransform, to_tuple
 from .functional import clipped, preserve_shape
 
-from skimage.exposure import match_histograms
+try:
+    from skimage.exposure import match_histograms
+except ImportError:
+    from skimage.transform import histogram_matching
```
Comment (Dipet) [inline albumentations/augmentations/domain_adaptation.py:13]: Functions has different names

**Review by Dipet — CHANGES_REQUESTED** [review]

File: albumentations/augmentations/domain_adaptation.py (hunk 1 above)
Comment (hhawary) [reply albumentations/augmentations/domain_adaptation.py:13]: Yes you are right it supposed to be from skimage.transform import match_histograms . my mistake.

**Comment by hhawary** [comment]
> What does this change do?
> 
> We do not use `histogram_matching` in the rest of the file.

please check https://github.com/scikit-image/scikit-image/blob/v0.15.x/skimage/transform/histogram_matching.py the function name is match_histograms

Starting from version 0.16.x they move this function match_histograms to this file https://github.com/scikit-image/scikit-image/blob/v0.16.x/skimage/exposure/histogram_matching.py

However they still have the same function in https://github.com/scikit-image/scikit-image/blob/v0.16.x/skimage/transform/histogram_matching.py

so for precautions i put it in a try except if in the future they remove it.

**Comment by Dipet** [comment]
Everything is working fine today, so these changes are not necessary.

**Pull request closed without merging**